### PR TITLE
add: mouse idle anim

### PR DESCRIPTION
**Pull request by ROdenFL**
## What Does This PR Do

Add 9 idle anims for mouse. For of them can be used by ai mice, some only for sentient one.

Mouses are immobolized, during the animation.

There is 20 seconds cd between animations.

Current lists for now:

Ai mouse and not sentient player: shake itself, washing up, sniffing, scratching. 

Sentient player: no sentience +  ass shake + dance + cig

Sprites by @FanTik-Ratge

## Why It's Good For The Game

Animations are made cool. It can be quite funny to them in-game.

## Images of changes

https://github.com/user-attachments/assets/ad8fffa8-1dc7-41d7-b6e7-f7d67f9eedae

*Every mouse in this film is older than 18

## Testing

Tested both sentient and unsentient mice.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
add: Mouse idle anims
/:cl:

### CODE REVIEW

**Comment by FanTik-Ratge** [comment]
👑🐀👍

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -231,3 +267,156 @@
...
+	emote("mdance", intentional = TRUE)
+
+/mob/living/simple_animal/mouse/proc/shakeass()
+	set name = "Shake Ass"
+	set desc = "Shake your ass"
+	set category = "Mouse"
+
+	emote("mshakeass", intentional = TRUE)
+
+/datum/emote/living/simple_animal/mouse/idle
```
Comment (lewcc) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:344]: If you add these emotes, they should go into a proper file just for emotes, which I think mice might already have?

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -37,6 +48,8 @@
 	can_collar = TRUE
 	gold_core_spawnable = FRIENDLY_SPAWN
 	var/chew_probability = 1
+	/// Is our mouse smart?
+	var/sentient = FALSE
```
Comment (lewcc) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:52]: Where is this getting set? Is this designed for adminbus?

Comment (ROdenFL) [reply code/modules/mob/living/simple_animal/friendly/mouse.dm:52]: In sentience event and potion from xeno

**Comment by PopeDaveThe3th** [comment]
Are the blatantly unnatural animations (flower, dance, thumbs up) player only? I love the idea of a player controlled mouse making someone think they're going nuts with 'em but they probably shouldn't be for NPC mice.

**Comment by ROdenFL** [comment]
> Are the blatantly unnatural animations (flower, dance, thumbs up, etc.) player only? I love the idea of a player controlled mouse making someone think they're going nuts with 'em but they probably shouldn't be for NPC mice.

 They're player only + sentience only

**Comment by ROdenFL** [comment]
Xeno potion plus sentience event

**Comment by 1080pCat** [comment]
i think the flower, and thumbs up one should be axed. they're way too unusual for mice, they're gaining humanoid hands, thats almost as big as them is a little too much. the hat dance, butt shake, and cig one all keeps it _semi_ realistic in the fact that they dont inexplicably gain a human hand.

Also do blob infected mice get the player controlled emotes? having a mouse shake its butt then explode into a blob would be hilarious

**Comment by ROdenFL** [comment]
> i think the flower, and thumbs up one should be axed. they're way too unusual for mice, they're gaining humanoid hands, thats almost as big as them is a little too much. the hat dance, butt shake, and cig one all keeps it _semi_ realistic in the fact that they dont inexplicably gain a human hand.
> 
> Also do blob infected mice get the player controlled emotes? having a mouse shake its butt then explode into a blob would be hilarious  

blobinfected is not sentience (can change it), but ass shake is non sentience emote. Thumb and flower can be moved to admin only or deleted.

**Comment by Christasmurf** [comment]
> > i think the flower, and thumbs up one should be axed. they're way too unusual for mice, they're gaining humanoid hands, thats almost as big as them is a little too much. the hat dance, butt shake, and cig one all keeps it _semi_ realistic in the fact that they dont inexplicably gain a human hand.
> > Also do blob infected mice get the player controlled emotes? having a mouse shake its butt then explode into a blob would be hilarious
> 
> blobinfected is not sentient (can change it), but ass shake is non sentience emote. Thumb and flower can be moved to admin only or deleted.

I'm not too sure on the hand emotes, they just seem way too out of place. The thugshaker, smoking and hat one I feel are good but should be only for sentient mice, and then the other 4 look good for all mice regardless of sentience.

**Comment by Denthamos** [comment]
> > > i think the flower, and thumbs up one should be axed. they're way too unusual for mice, they're gaining humanoid hands, thats almost as big as them is a little too much. the hat dance, butt shake, and cig one all keeps it _semi_ realistic in the fact that they dont inexplicably gain a human hand.
> > > Also do blob infected mice get the player controlled emotes? having a mouse shake its butt then explode into a blob would be hilarious
> > 
> > 
> > blobinfected is not sentient (can change it), but ass shake is non sentience emote. Thumb and flower can be moved to admin only or deleted.
> 
> I'm not too sure on the hand emotes, they just seem way too out of place. The thugshaker, smoking and hat one I feel are good but should be only for sentient mice, and then the other 4 look good for all mice regardless of sentience.

Design wise I am objecting to all but the top right, middle right, middle left, and bottom middle.  The rest are too memey.

**Comment by ROdenFL** [comment]
> > > > i think the flower, and thumbs up one should be axed. they're way too unusual for mice, they're gaining humanoid hands, thats almost as big as them is a little too much. the hat dance, butt shake, and cig one all keeps it _semi_ realistic in the fact that they dont inexplicably gain a human hand.
> > > > Also do blob infected mice get the player controlled emotes? having a mouse shake its butt then explode into a blob would be hilarious
> > > 
> > > 
> > > blobinfected is not sentient (can change it), but ass shake is non sentience emote. Thumb and flower can be moved to admin only or deleted.
> > 
> > I'm not too sure on the hand emotes, they just seem way too out of place. The thugshaker, smoking and hat one I feel are good but should be only for sentient mice, and then the other 4 look good for all mice regardless of sentience.
> 
> Design wise I am objecting to all but the top right, middle right, middle left, and bottom middle.  The rest are too memey/out of place for a regular mouse.

I made some changes. Mice animations with hands was transfered to new admin mouse. I need to test code after last commit and change desc.

**Comment by ROdenFL** [comment]
Normal sentienceless mouse has only "idle*" animations

**Comment by Denthamos** [comment]
> > > > > i think the flower, and thumbs up one should be axed. they're way too unusual for mice, they're gaining humanoid hands, thats almost as big as them is a little too much. the hat dance, butt shake, and cig one all keeps it _semi_ realistic in the fact that they dont inexplicably gain a human hand.
> > > > > Also do blob infected mice get the player controlled emotes? having a mouse shake its butt then explode into a blob would be hilarious
> > > > 
> > > > 
> > > > blobinfected is not sentient (can change it), but ass shake is non sentience emote. Thumb and flower can be moved to admin only or deleted.
> > > 
> > > 
> > > I'm not too sure on the hand emotes, they just seem way too out of place. The thugshaker, smoking and hat one I feel are good but should be only for sentient mice, and then the other 4 look good for all mice regardless of sentience.
> > 
> > 
> > Design wise I am objecting to all but the top right, middle right, middle left, and bottom middle.  The rest are too memey/out of place for a regular mouse.
> 
> I made some changes. Mice animations with hands were transfered to new admin mouse. I need to test code after last commit and change desc.

As an admin on the design team, I _still_ object to such animations being in the game at all even admin only wise.  Such has no purpose here on this server.

**Comment by S34NW** [comment]
This is currently on track to be closed with 1 sprite team objection and 3 design team objections. I'll give you a few days to figure out how to proceed before closing if no changes are made.

The comments against are:

- meme ones are a no
- The ones with human hands have been moved to admin-only, seemingly with an additional "jetpack" version joining them in admin-only. I would prefer they simply not be in the PR at all. Personally I think the other 'meme' ones are fine for sentient mice only.
- Agreeing with first comment. Other anims are fine
- I am objecting on all but top right, middle right, middle left, and bottom middle. The rest of them have an objection from me on them.

**Comment by Qwertytoforty** [comment]
Still not fine with shakeass, you have code for some emotes like smoking still left

**Comment by ROdenFL** [comment]
I don't see this pr without ratass. I can remove any of emotes, expect this one. I guess, it is easier to close a pr.

**Comment by farie82** [comment]
Sorry for the delay, but we've decided to close this PR for the reasons described in an earlier comment. Most of the content is fine, so a future PR can be reopened if it took the objections into account.

**Pull request closed without merging**